.nuspec/Xamarin.Forms.targets

```diff
@@ -66,35 +42,42 @@
 			Condition="'$(_XFTasksAbi)' != '$(_XFTasksExpectedAbi)'" />
 	</Target>
 
-	<Target Name="_PreXamlG">
-		<MakeDir Directories="$(IntermediateOutputPath)"/>
-	</Target>
-  
-	<Target Name="_CollectXamlFiles">
-		<ItemGroup>
-			<_XamlResources Include="@(EmbeddedResource)" Condition="'%(Extension)' == '.xaml' AND '$(DefaultLanguageSourceExtension)' == '.cs'"/>
-		</ItemGroup>
-		<FixedCreateCSharpManifestResourceName ResourceFiles="@(_XamlResources)" RootNamespace="$(RootNamespace)">
-			<Output TaskParameter="ResourceFilesWithManifestResourceNames" ItemName="XamlFiles" />
-		</FixedCreateCSharpManifestResourceName>
-		<ItemGroup>
-			<XamlGFiles Include="@(XamlFiles->'$(IntermediateOutputPath)%(ManifestResourceName).g$(DefaultLanguageSourceExtension)')"/>
-			<Compile Include="@(XamlGFiles)"/>
-			<FileWrites Include="@(XamlGFiles)"/>
-		</ItemGroup>
+	<Target Name="_ValidateMSBuild" BeforeTargets="_CheckForInvalidConfigurationAndPlatform">
+		<Error
+			Text="Xamarin.Forms require msbuild. xbuild is no longer supported."
+			Code="XF003"
+			Condition="'$(IsXbuild)' == 'true'" />
```

Comment:
`IsXBuild`

that's not because it's case insensitive that you can type whatever you want